### PR TITLE
[smartswitch] Update smartswitch golden config

### PR DESCRIPTION
**Pull request by wen587**
### Description of PR

Summary: Update SmartSwitch Golden Config for dhcp and gnmi
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
More config table need to be updated for SmartSwitch
#### How did you do it?
Update SmartSwitch Golden Config for dhcp and gnmi
#### How did you verify/test it?
E2E

### CODE REVIEW

**Review by ganglyu — APPROVED** [review]

**Review by ganglyu — APPROVED** [review]

File: ansible/library/generate_golden_config_db.py [hunk 1]
```diff
@@ -101,10 +101,31 @@ def generate_smartswitch_golden_config_db(self):
...
+                or "dhcp_server" not in ori_config_db["FEATURE"] \
+                or "dhcp_relay" not in ori_config_db["FEATURE"]:
+            return "{}"
+        ori_config_db["FEATURE"]["dhcp_server"]["state"] = "enabled"
+        ori_config_db["FEATURE"]["dhcp_relay"]["state"] = "enabled"
+
+        # Generate INTERFACE table for EthernetBPXX
+        if "PORT" not in ori_config_db or "INTERFACE" not in ori_config_db:
+            return "{}"
+        for i in range(8):
```
Comment (qiluo-msft) [inline ansible/library/generate_golden_config_db.py:116]: What is the meaning of 8? Is it always 8?

Comment (wen587) [reply ansible/library/generate_golden_config_db.py:116]: For cisco, it is alwasy 8.

File: ansible/library/generate_golden_config_db.py [hunk 2]
```diff
@@ -101,10 +101,31 @@ def generate_smartswitch_golden_config_db(self):
...
+            port_key = "Ethernet-BP{}".format(i)
+            interface_key = "Ethernet-BP{}|18.{}.202.0/31".format(i, i)
```
[the remark this answers is not in the document]
Comment (wen587) [reply ansible/library/generate_golden_config_db.py:118]: I don't see it in topo.yml. Shared some chat info through teams.

File: ansible/library/generate_golden_config_db.py (hunk 1 above)
Comment (qiluo-msft) [reply ansible/library/generate_golden_config_db.py:116]: Seems it belongs to topo.yml. Suggest extend current topo infra.

File: ansible/library/generate_golden_config_db.py [hunk 3]
```diff
@@ -26,6 +26,14 @@
 TEMP_SMARTSWITCH_CONFIG_PATH = "/tmp/smartswitch.json"
 DUMMY_QUOTA = "dummy_single_quota"
 
+smartswitch_hwsku_config = {
```
Comment (wen587) [inline ansible/library/generate_golden_config_db.py:29]: Hi @qiluo-msft Updated the code to make people to easy to udpate their port and inferface ip based on hwsku.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**